content/projects/gsoc/2020/project-ideas/jenkins-distribution-customize-service.adoc

```diff
@@ -0,0 +1,52 @@
+---
+layout: gsocprojectidea
+title: "Custom Jenkins distribution build service"
+goal: "Provide an out of the box solution for packaging Jenkins distributions as WAR files or Docker images"
+category: Tools
+year: 2020
+status: draft
+skills:
+- Java
+- Web Development
+- Docker
+mentors:
+- "linuxsuren"
+links:
+  gitter: "jenkinsci/custom-distribution-build-service"
```

Comment:
rename it to match the proposal? We could also use an existing entity like Platform SIG for now

Reply:
Agree, good idea. Let's reuse the Platform SIG.